src/Extensions/SearchServiceFileExclusionExtension.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace SilverStripe\Forager\Extensions;
+
+use SilverStripe\Core\Config\Configurable;
+use SilverStripe\Core\Extension;
+
+/**
+ * Add the ability to exclude specific file extensions
+ */
+class SearchServiceFileExclusionExtension extends Extension
+{
+
+    use Configurable;
+
+    public function canIndexInSearch(): bool
+    {
+        $owner = $this->getOwner();
+
+        $excludeFileTypes = $this->config()->get('exclude_file_extensions') ?? [];
```

Comment:
the docs mention putting the config on File but this is looking for config on SearchServiceFileExclusionExtension - did you want to check $owner instead